agave_app/Controls.cpp

```diff
@@ -1,18 +1,21 @@
 #include "Controls.h"
 
 #include <QColorDialog>
+#include <QLabel>
 #include <QPaintEvent>
 #include <QPainter>
 #include <QPainterPath>
 #include <QtGlobal>
 
 QColorPushButton::QColorPushButton(QWidget* pParent)
   : QPushButton(pParent)
-  , m_Margin(5)
+  , m_Margin(4)
   , m_Radius(4)
   , m_Color(Qt::gray)
 {
   setText("");
+  // deal with a macos style bug which causes this button to misalign in some situations
+  setAttribute(Qt::WA_LayoutUsesWidgetRect);

```

Comment:
Is this still a bug with the fusion style?

Reply:
It's not a problem with fusion, but it still doesn't hurt to leave it in.  If we ever switch back it would suddenly break something.